jangjuseong/controllers/postController.js

```diff
@@ -0,0 +1,48 @@
+const postService = require('../services/postService');
+
+const createPost = async (req, res) => {
+  const { title, content, postImgUrl, userId } = req.body;
+
+  await postService.postUp(title, content, postImgUrl, userId);
+
+  return res.status(200).json({ message: 'postCreated' });
+};
+
+const getPost = async (req, res) => {
```

Comment:
전체 게시글을 조회한다는 의미로서 복수형으로 함수명을 지으면 더 좋을 것 같습니다!

```suggestion
const getPosts = async (req, res) => {
```